Date: Mon, 07 Jan 2008 00:16:46 +0500
From: Valentine Sinitsyn <valentine.sinitsyn@usu.ru>
To: kde-networkmanager@kde.org
Subject: Autostart entry

Hi everyone,

It appears that KNM-current (and 0.2, too) fails to autostart regardless 
of the setting in the config dialog (at least on my system, which is 
Arch Linux Core Dump with almost vanilla KDE 3.5.8). The problem is KNM 
autostart .desktop entry isn't installed where it should be 
($KDEPREFIX/share/autostart). The patch is trivial (see attached file) 
and was shamelessly borrowed from kdebluetooth sources.

...
Best wishes & Happy New Year!
Valentine Sinitsyn

Index: Makefile.am
===================================================================
--- knetworkmanager/knetworkmanager/src/Makefile.am	(revision 757371)
+++ knetworkmanager/knetworkmanager/src/Makefile.am	(working copy)
@@ -66,6 +66,10 @@
 # this is where the desktop file will go 
 xdg_apps_DATA = knetworkmanager.desktop
 
+# Autostart entry
+autostartdir = $(datadir)/autostart
+autostart_DATA = knetworkmanager.desktop
+
 # this is where the shell's XML-GUI resource file goes
 shellrcdir = $(kde_datadir)/knetworkmanager
 shellrc_DATA = knetworkmanagerui.rc

